helm/cluster/templates/_helpers.tpl

```diff
@@ -299,12 +299,42 @@ Where `data` is the data to hash and `global` is the top level scope.
 {{- if $appCatalog }}
 {{- $appCatalog }}
 {{- else if $renderWithoutReleaseResource -}}
-fake-app-catalog-from-offline-cluster-chart-rendering
+{{- printf "fake-app-catalog-from-offline-cluster-chart-rendering" }}
 {{- else }}
-{{- fail (printf "Application not found in Release/%s: %s" (($.GiantSwarm.Release).metadata).name $.appName) }}
+{{- printf "no-catalog-found" }}
```

Comment:
I think we can leave this as is, because checking if the app is in the release is a prerequisite to me, so this function shouldn't even be called if the app is not in the release, right?